data/json/items/furniture.json

```diff
@@ -0,0 +1,118 @@
+[
+  {
+    "type": "GENERIC",
+    "id": "chair_wood",
+    "name": { "str": "wooden chair" },
+    "description": "A simple wooden chair, with four legs, a seat, and a back.  It's nice to rest your feet for once, and if coupled with a suitable table, you could eat a meal properly and almost pretend that things were normal again.",
+    "symbol": "#",
+    "color": "brown",
+    "flags": [ "NO_SALVAGE" ],
+    "price": 3000,
+    "price_postapoc": 250,
+    "longest_side": "95 cm",
+    "volume": "112 L",
+    "weight": "12 kg",
+    "material": [ "wood" ],
+    "to_hit": -1,
+    "bashing": 5
+  },
+  {
+    "type": "GENERIC",
+    "id": "armchair",
+    "name": { "str": "armchair" },
+    "description": "A simple upholstered chair with armrests.  Soft and fairly warm, it definitely beats the ground for sleeping on, though not by much.",
```

Comment:
Shouldn't this have a ``"comfort"`` value so it can beat the ground for sleeping on 🤔

Reply:
Can items have comfort value? If they do that's news to me, I've never messed around with clothing, but if so then you're certainly right. I'll take a look around when I'm free later today.

Reply:
Okay, I looked at this again, items do not appear to have comfort values. I could give it the SLEEP_AID flag but I'll ask around for opinions before doing that.